src/equations/serre_green_naghdi_1d.jl

```diff
@@ -828,7 +827,7 @@ Return the modified total energy of the primitive variables `q_global` for the
 [`SerreGreenNaghdiEquations1D`](@ref).
 It contains an additional term containing a
 derivative compared to the usual [`energy_total`](@ref) modeling
-non-hydrostatic contributions. The [`energy_total_modified`](@ref)
+non-hydrostatic contributions. The `energy_total_modified`
```

Comment:
This is the docstring for `energy_total_modified`, so it doesn't really make sense to link to it, I think

Reply:
I agree. Thanks!